src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -107,7 +107,8 @@ case class AgoraEntity(namespace: Option[String] = None,
                        entityType: Option[AgoraEntityType.EntityType] = None,
                        id: Option[ObjectId] = None,
                        methodId: Option[ObjectId] = None,
-                       method: Option[AgoraEntity] = None) {
+                       method: Option[AgoraEntity] = None,
+                       managers: Seq[String] = Seq()) {
```

Comment:
I had that at first, but @davidangb suggested just making it a seq that's set as empty instead of an option set at none